compiler/il/symbol/OMRResolvedMethodSymbol.hpp

```diff
@@ -130,6 +130,11 @@ class OMR_EXTENSIBLE ResolvedMethodSymbol : public TR::MethodSymbol
 
    void addVariableSizeSymbol(TR::AutomaticSymbol *s);
 
+   /*
+    * Get known object index of a parameter if there is any known object information available
```

Comment:
can you describe what slot is and how counting works for a call since OMR is not Java and a slot can be an ambiguous concept...

Reply:
changed `slot` to `argIndex` and added an explanation.

Reply:
Thanks for pointing that out! changed from `slot` to `ordinal` and added explanation.